app/Domain/Integrations/Mappers/UpdateContactInfoMapper.php

```diff
@@ -68,9 +68,9 @@ public static function map(UpdateContactInfo $updateContactInfo, string $integra
             $contacts[] = $contact;
         }
 
-        $newLastName = $updateContactInfo->input('newContributorLastName');
-        $newFirstName = $updateContactInfo->input('newContributorFirstName');
-        $newEmail = $updateContactInfo->input('newContributorEmail');
+        $newLastName = $request->input('newContributorLastName');
+        $newFirstName = $request->input('newContributorFirstName');
+        $newEmail = $request->input('newContributorEmail');
```

Comment:
I'll also move this logic to a separate `POST` call to `/integrations/{id}/contacts`

Reply:
We already have `IntegrationController::updateContacts`, so I think that is fine?

Reply:
Yes, it works. We can send `newContributorLastName`, ... to the `updateContacts`.
Wouldn't it be cleaner if the creation of a new contact was separate?

Reply:
I think I now understand de confusion. This is a mix between new contacts and updated contacts. I think it was designed that way to better match the UI that has a big editable list.

If I would create an endpoint without taking those requirements into account I would indeed make a store and update endpoint. The consumer of those endpoints needs to call the store and update one or several times.

Reply:
In the frontend we are changing the ui atm to use dialogs/modals for creating a new contact for example. There it makes sense to fire a POST request solely for that new contact.

Reply:
Yes, I would also prefer a clear `POST` endpoint to create new contacts and a `PUT` endpoint to update existing contacts.

Reply:
But to limit the scope of this PR it may be best to do this in a follow-up pull request?